app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -34,11 +34,13 @@
 public class ListMessageAdapter extends RecyclerView.Adapter<ListMessageAdapter.ViewHolder> {
 
     private Context context;
```

Comment:
When starting the app without any cache / data (clear storage in android settings), then the popup for restart is visible, without any change in the settings. 

I also had a crash the first time I've opened the settings page, but I cannot reproduce it anymore :/.

[Peek 2022-10-09 15-09.webm](https://user-images.githubusercontent.com/14895212/194758626-fffd70d4-6380-47ae-82ac-c85b6d3745d8.webm)

Reply:
This is weird, I will investigate it and (hopefully) fix it soon.